chapters/robustness.adoc

```diff
@@ -65,3 +65,11 @@ For the case of accessing an image LOD outside the supported range, with `robust
 Without the link:https://registry.khronos.org/vulkan/specs/1.3-extensions/html/vkspec.html#features-nullDescriptor[nullDescriptor] feature enabled, when updating a `VkDescriptorSet`, all the resources backing it must be non-null, even if the descriptor is statically not used by the shader. This feature allows descriptors to be backed by null resources or views. Loads from a null descriptor return zero values and stores and atomics to a null descriptor are discarded.
 
 The `nullDescriptor` feature also allows accesses to vertex input bindings where `vkCmdBindVertexBuffers::pBuffers` is null.
+
+== VK_EXT_pipeline_robustness
+
+Because for some implementations robustness can come at a performance cost, the `VK_EXT_pipeline_robustness` extension was added to allow a developers to request robustness on a per-pipeline stage basis.
+
+At `VkPipeline` creation time the `VkPipelineRobustnessBufferBehaviorEXT` and `VkPipelineRobustnessImageBehaviorEXT` can set which robustness for the resources of the pipeline.
```

Comment:
The names of the behavior enum types are a low-level detail - at the 10K-foot level, what you have to do to use this extension is to pass a struct to the pipeline creation call. So, suggest
```suggestion
At `VkPipeline` creation time one or more `VkPipelineRobustnessCreateInfoEXT` structures can be passed to specify the desired robustness behavior of accesses to buffer, image, and vertex input resources, either for the pipeline as a whole or on a per-pipeline-stage basis.
```